cloud-sql/postgres/client-side-encryption/snippets/cloud_sql_connection_pool.py

```diff
@@ -19,6 +19,9 @@
 def init_tcp_connection_engine(
     db_user: str, db_pass: str, db_name: str, db_host: str
 ) -> sqlalchemy.engine.base.Engine:
+    """
```

Comment:
One liner docstrings should have the quotes on the same line. And still use triple quotes. Don't know why, but that's what our style guide says.